linera-witty/tests/reentrancy.rs

```diff
@@ -464,9 +347,13 @@ where
     InstanceFactory::Instance: InstanceForEntrypoint,
     <<InstanceFactory::Instance as Instance>::Runtime as Runtime>::Memory:
         RuntimeMemory<InstanceFactory::Instance>,
-    ExportedOperations: ExportTo<InstanceFactory::Builder>,
+    ExportedOperations<InstanceFactory::Caller<'static>>: ExportTo<InstanceFactory::Builder>,
 {
-    let instance = factory.load_test_module::<ExportedOperations>("reentrancy", "operations");
+    let instance = factory
+        .load_test_module::<ExportedOperations<InstanceFactory::Caller<'static>>>(
```

Comment:
Maybe we should have picked the other choice in the last PR:
```
factory.load_test_module("reentrancy", "operations", ExportedOperations::export_to)
```

Anyways, you should be able to write `load_test_module::ExportedOperations<_>(...)`.

Reply:
Yeah, maybe it could have simplified things a bit now :thinking: 

Thanks for the tip to omit the type parameter! That's a lot cleaner :grin: